OrbitBase/File.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
```

Comment:
The name of this file is way too generic (`File.h`/`File.cpp`). If for now it only contains this one function, for me it's also fine to just call it `ReadFileToString.cpp`/`.h`, but feel free to come up with something better, as long as it's more specific.

Reply:
done.